chains/manager.go

```diff
@@ -1321,6 +1324,33 @@ func (m *manager) registerBootstrappedHealthChecks() error {
 	if err := m.Health.RegisterHealthCheck("bootstrapped", bootstrappedCheck, health.ApplicationTag); err != nil {
 		return fmt.Errorf("couldn't register bootstrapped health check: %w", err)
 	}
+
+	// If the node is not lite syncing the Primary Network, there is no reason
+	// to shutdown if it is a validator.
+	if !m.LiteSyncPrimaryNetwork {
+		return nil
+	}
+
+	liteSyncCheck := health.CheckerFunc(func(ctx context.Context) (interface{}, error) {
+		// Note: The health check is skipped during bootstrapping to allow a
+		// node to sync the network even if it was previously a validator.
+		if !m.IsBootstrapped(constants.PlatformChainID) {
+			return "node is currently bootstrapping", nil
+		}
+		if !validators.Contains(m.Validators, constants.PrimaryNetworkID, m.NodeID) {
+			return "node is not a primary network validator", nil
+		}
+
+		m.Log.Fatal("node is a primary network validator. Shutting down...",
+			zap.Error(errNoLiteSyncForValidators),
+		)
+		go m.ShutdownNodeFunc(1)
+		return "node is a primary network validator", errNoLiteSyncForValidators
+	})
+
+	if err := m.Health.RegisterHealthCheck("validation", liteSyncCheck, health.ApplicationTag); err != nil {
```

Comment:
nit: not sure `validation` is the right name for this?

Reply:
I feel like it makes sense... The check is verifying that the node is not validating